Source/Particles/Collision/BackgroundMCCCollision.H

```diff
@@ -0,0 +1,75 @@
+/* Copyright 2021 Roelof Groenewald
+ *
+ * This file is part of WarpX.
+ *
+ * License: ????
```

Comment:
Something should go in here.

Reply:
Just the regular one we use for all files:
```suggestion
 * License: BSD-3-Clause-LBNL
```